packages/picker/src/Picker.ts

```diff
@@ -319,9 +323,8 @@ export class PickerBase extends SizedMixin(Focusable) {
         return html`
             <button
                 aria-haspopup="true"
-                aria-controls="popover"
                 aria-expanded=${this.open ? 'true' : 'false'}
-                aria-label=${ifDefined(this.label || undefined)}
+                aria-labelledby="button label"
```

Comment:
Referencing _self_ chooses to `aria-label` that will be managed by `sp-field-label` elements.

Referencing `label` points to the current "value" of the picker.